Makefile

```diff
@@ -35,6 +35,14 @@ ifneq ($(GLUON_BRANCH),)
   GLUON_AUTOUPDATER_ENABLED ?= 1
 endif
 
+ifneq ($(GLUON_FEATURES)$(GLUON_FEATURES_standard)$(GLUON_FEATURES_tiny),)
+  $(warning *** Warning: GLUON_FEATURES has been deprecated, please use the image-customization framework instead.)
```

Comment:
Maybe we should just write "image-customization file" rather than framework, like the docs do

Reply:
Also, deprecated implies that it still works, which is not the case - it should be called obsolete instead.

It might also make sense to make this a hard error like obsolete `site.conf` entries, as updating the site is required to keep things working as intended.